src/v/s3/signature.cc

```diff
@@ -116,7 +116,7 @@ inline void append_hex_utf8(ss::sstring& result, char ch) {
     result.append(h.data(), h.size());
 }
 
-static ss::sstring uri_encode(const ss::sstring& input, bool encode_slash) {
+ss::sstring uri_encode(const seastar::sstring& input, bool encode_slash) {
```

Comment:
Why this change is needed? I mean the namespace, not the static.

Reply:
I relied on clion to refactor and it messed up the namespace during refactor, will fix.

Reply:
fixed